lib/pbench/agent/tool_meister.py

```diff
@@ -431,7 +430,7 @@ def __init__(self, pbench_bin, params, redis_server, logger):
         # tools for storing their collected data.
         self._tool_dir = None
         # The temporary directory to use for capturing all tool data.
-        self._tmp_dir = os.environ.get("_PBENCH_TOOL_MEISTER_TMP", "/var/tmp")
+        self._tmp_dir = os.environ["pbench_tmp"]
```

Comment:
This prevents us from using `/var/tmp` during the test runs, and brings the Tool Meister in-line with the other code using the `${pbench_run}/tmp` directory.